<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winetricks</class>
 <widget class="QDialog" name="winetricks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>winetricks@q4wine</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <widget class="QListWidget" name="lstMain">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="cmdInstall">
     <property name="text">
      <string>Install</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="cmdExit">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Click on item to show help here</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="cmdInstWinetricks">
     <property name="text">
      <string>Install Winetricks</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
